package.json

```diff
@@ -18,13 +18,16 @@
     "@types/react-dom": "18.0.4",
     "@types/react-table": "7.7.12",
     "axios": "0.27.2",
+    "chart.js": "^3.8.2",
+    "chartjs-adapter-date-fns": "^2.0.0",
     "date-fns": "2.29.1",
     "history": "5.3.0",
     "i18next": "21.8.14",
     "immutability-helper": "3.1.1",
     "loglevel": "1.8.0",
     "react": "18.2.0",
     "react-beautiful-dnd": "13.1.0",
+    "react-chartjs-2": "^4.3.1",
```

Comment:
Should these dependencies be pinned?

Reply:
Also noticed there's a new minor version of `chart.js`, 3.9.1. Worth updating to give it a try